cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -125,6 +125,8 @@ const (
 	indexStateEmpty       indexState = "empty"        // index is empty (empty repo)
 )
 
+const ServiceName = "zoekt-sourcegraph-indexserver"
```

Comment:
this doesn't need to be exported. Additionally this is only read in one place, just inline the string there rather than having indirection to the value.

Reply:
Inlined!